resources/views/pages-legacy/forumposthistory.blade.php

```diff
@@ -58,12 +59,18 @@
 
     $isShowAbsoluteDatesPreferenceSet = $websitePrefs && BitSet($websitePrefs, UserPreference::Forum_ShowAbsoluteDates);
 
+    $fetchedUserCache = [];
+
     foreach ($recentPosts as $topicPostData) {
         $postMessage = $topicPostData['ShortMsg'];
         if ($topicPostData['IsTruncated']) {
             $postMessage .= '...';
         }
-        $postAuthor = $topicPostData['Author'];
+        
+        $authorId = $topicPostData['author_id'];
```

Comment:
```
Undefined array key "author_id"
```

Reply:
Thanks, I wasn't seeing this during testing. It took a little bit of trial and error but I managed to repro. Resolved in latest.